README.md

```diff
@@ -1 +1,26 @@
-# community
\ No newline at end of file
+# Human Cell Atlas Data Coordination Platform community
+
+** Work in Progress **
+
+Welcome to the community!
+
+## Communicating
+
```

Comment:
I expect that this section will be replaced with a link to datahumancellatlas.org which will become the _community entrance_ to HCA DCP.

Reply:
Yes and then also maybe specifically the about us and contact us sections.